Cargo.toml

```diff
@@ -22,7 +22,7 @@ default = [
 
 [dependencies]
 winit = {version = "0.25.0", features = ["web-sys"]}
-bevy = { version="0.5.0", default-features=false }
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features=false }
```

Comment:
as you are changing bevy version here please remove whole `[patch.crates-io]` section below